frontend/src/metabase/public/containers/PublicOrEmbeddedDashboard/PublicOrEmbeddedDashboard.tsx

```diff
@@ -208,33 +212,35 @@ const PublicOrEmbeddedDashboardInner = ({
   });
 
   return (
-    <PublicOrEmbeddedDashboardView
-      dashboard={dashboard}
-      hasNightModeToggle={hasNightModeToggle}
-      isFullscreen={isFullscreen}
-      isNightMode={isNightMode}
-      onFullscreenChange={onFullscreenChange}
-      onNightModeChange={onNightModeChange}
-      onRefreshPeriodChange={onRefreshPeriodChange}
-      refreshPeriod={refreshPeriod}
-      setRefreshElapsedHook={setRefreshElapsedHook}
-      selectedTabId={selectedTabId}
-      parameters={parameters}
-      parameterValues={parameterValues}
-      draftParameterValues={draftParameterValues}
-      setParameterValue={setParameterValue}
-      setParameterValueToDefault={setParameterValueToDefault}
-      dashboardId={dashboardId}
-      background={background}
-      bordered={bordered}
-      titled={titled}
-      theme={theme}
-      hideParameters={hideParameters}
-      navigateToNewCardFromDashboard={navigateToNewCardFromDashboard}
-      slowCards={slowCards}
-      cardTitled={cardTitled}
-      downloadsEnabled={downloadsEnabled}
-    />
+    <LocaleProvider locale={locale}>
```

Comment:
I deliberately pass `locale` from the parent component so I could have `LocaleProvider` wrapped `PublicOrEmbeddeDashboardView` similar to how I also have it wrapped `PublicOrEmbeddedQuestionView`.

I could use `LocaleProvider` in the parent component, but I think putting it here would make it easier to compare between the 2 similar view components.

Reply:
The important thing is that it's rendered right away (not after a loading) so that it fires the requests asap. It looks that where it is now it works like that so it's fine